homeworks/artem.sokur_Grimdvl/PokemonProject/login.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://fonts.google.com/share?selection.family=Roboto:wght@100;300;700">
+        <link rel="stylesheet" href="style/style.css">
+        <link rel="stylesheet" href="style/login.css">
+        <title>PokemonProject</title>
+    </head>
+    <body>
+        <header>
+            <div class="container" style="background-color: #333;">
+                <section class="head">
+                    <a class="head__logo" href="https://www.mastersacademy.education/" target="_blank">
+                        <img class="head__logo-img" src="img/icons/logo-ma.svg" alt="masters-academy">
+                        <span class="head__logo-title">MASTERS ACADEMY</span>
+                    </a>
+    
+                    <nav class="head__navigation">
+                        <a class="head__navigation-item" href="pokemons.html">Pokemons</a>
+                        <a class="head__navigation-item" href="index.html">About</a>
+                        <a class="head__navigation-item active" href="#">Login</a>
+                    </nav>
+                </section>
+            </div>
+        </header>
+
+        <main>
+            <section class="login">
+                <h1 class="login__title">Login page</h1>
+                <form class="login__form" action="success.html">
+                    <div class="login__form-input">
+                        <label for="email">Email</label>
+                        <input placeholder="Enter your email address" name="email" type="email" required>
+                    </div>
+                    <div class="login__form-input">
+                        <label for="password">Password</label>
+                        <input placeholder="Enter your password" name="password" type="password" required minlength="4">
+                    </div>
+                    <div class="login__form-checkbox">
+                        <input name="checkbox" type="checkbox" id="checkbox" required>
+                        <label for="checkbox">I'm not a robot</label>
+                    </div>
+                    <button type="submit" class="login__form--submit">Login</button>
```

Comment:
need to fix margin!
<img width="831" alt="Screenshot 2023-10-25 at 18 59 06" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/115080771/6ba92920-42f6-493b-aba0-192aa926cf58">